src/utils/api.js

```diff
@@ -3,8 +3,8 @@ import axios from "axios";
 const newsApi = axios.create({
     baseURL: "https://be-portfolio-project.herokuapp.com/api"
 })
-export const getArticles = () => {
-    return newsApi.get('/articles').then((res) => {
+export const getArticles = (sortedBy, orderedBy) => {
```

Comment:
You will be able to refactor this to use the axios params object which will append queries such as `sort)by` and `order` if they are defined. You will not need to provide these options in the request endpoint.